hooks/index.js

```diff
@@ -1 +1,2 @@
 export * from './use-scroll-position'
+export * from './copy-to-clipboard'
```

Comment:
let's stick `copy-to-clipboard` in our `util` dir. the `hooks` dir is more reserved for these `use*` functions that hook into React's rendering lifecycle